src/components/patterns/ContentFeature.js

```diff
@@ -15,7 +15,7 @@ CustomImage.defaultProps = {
 
 export default ({ image, children, title, flexDirection = 'left' }) => (
   <Flex
-    alignContent={'center'}
```

Comment:
The last version of rebass/styled-system renamed `align` into `alignContent` for Flexbox props:
https://github.com/jxnblk/styled-system#flexbox